src/lib/pipeline.ml

```diff
@@ -84,9 +87,14 @@ type _ t =
   | Bam_pair: bam t * bam t -> bam_pair t
   | Somatic_variant_caller: somatic Variant_caller.t * bam_pair t -> vcf t
   | Germline_variant_caller: germline Variant_caller.t * bam t -> vcf t
+  | Rna_hla_typer: fastq_sample t -> hla_types t
```

Comment:
I'm not so certain. There `Somatic_variant_caller` obviously does not refer to a specific tool. There seems to be a distinction between aligners and variant callers, where we use names in the first and what they do in the second. I think what a node of a pipeline does makes more sense, than the tool. What tool performs the specific step could be specified differently. Since `seq2HLA` is (at the moment) the only RNA typing tool. It is used by default.